src/multiplayer-servers/getting-started/setup-third-party-oauth.md

```diff
@@ -0,0 +1,122 @@
+# Setting Up Third-Party OAuth Integration
+
+## Overview
+
+GameFabric supports Third-Party OAuth Integration with every OAuth-compatible Identiy Provider, allowing you to reuse your existing authentication flows, without giving GameFabric access to your sensitive authentication data.
+
+## Prerequisites
+
+Before setting up a Third-Party OIDC Provider, ensure the following conditions are met:
+
+- You have access to an active GameFabric installation with access through the default user.
+- You have an OIDC ([OpenID Connect](https://openid.net/developers/how-connect-works/)) provider, including a **Client ID** and **Client Secret**.
+
+## Setting up a new OIDC Provider
+
+### Step 1: Add a New Provider
```

Comment:
This step numbering is inconsistent with the rest of the documentation. In general documentation should be consistent. This includes using bullet points over numbered lists.